packages/runtime/container-runtime/package.json

```diff
@@ -255,6 +255,7 @@
 			"Class_LocalFluidDataStoreContextBase": {
 				"forwardCompat": false
 			}
-		}
+		},
+		"entrypoint": "legacy"
```

Comment:
I'm not sure I understand the implications - does this mean that the type testing for `@internal` APIs will be dropped from this package, because we will only be typetesting for legacy-and-up?

Reply:
Ah, I think this is the case based on your updated documentation.